iBudget/static/src/tabs/Groups.vue

```diff
@@ -1,27 +1,49 @@
 <template>
   <div class="content">
       <div class="text">
-      <h1>There will be groups...</h1>
+      <p>There are your groups: </p>
+          <ul id="groups">
+              <li v-for="(content,group) in cur_balance" >
+                  {{group}}
+                  <ul>
+                      <li v-for="(item,value) in content" >
+                          {{value}} : {{item}}
+                      </li>
+                  </ul>
+              </li>
+          </ul>
     </div>
   </div>
-
 </template>
 
 <script>
+    import axios from 'axios';
     export default {
-        name: "Groups"
-    }
+        name: "Groups",
+        data (){
+            return{
+                cur_balance : []
+            }} ,
+        created() {
+            axios.get('api/v1/group/')
+                .then(response => {
+                    this.cur_balance = response.data
+                })
+                .catch(e => {
+                    this.errors.push(e)
+                })
+        }}
 </script>
 
 <style scoped>
```

Comment:
Indentation
In `script` you have indentation with 4 spaces.
Here you have 2 spaces in one part and 6 in other.
also `.text` is not child of `.content`

Also it is better to have space between `tag\#id\.class` of element and `{`